Makefile

```diff
@@ -8,6 +8,9 @@ install:
 		./scripts/install-vim $$vim; \
 	done
 
+	# only need to install the go tools once...
```

Comment:
Actually this isn't true, as it installs the Go tools to a temporary GOPATH in the Vim's install directory, so it needs to be installed for every Vim. This is why it was in the `install-vim` script.